### PR TITLE
Create localization.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Moved Localization to handbook.

### CODE REVIEW

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thanks! Please add to SUMMARY.md, below "help wanted tickets"

**Review by jasonblais — DISMISSED** [review]

File: contributors/contributors/localization.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# Localization
+
+The goal of the localization process is to consistently produce high quality translations release after release through collaboration between the international community, translators, project managers, language leads, and the core team.
+
+## Translation Process
+
+If you're interested in contributing translations to Mattermost, please join the [Mattermost localization channel to discuss] (https://community.mattermost.com/core/channels/localization). Below are descriptions of the overall localization process.
```
Comment (amyblais) [inline contributors/contributors/localization.md]: Link here is broken

Comment (justinegeffen) [inline contributors/contributors/localization.md]:
```suggestion
If you're interested in contributing translations to Mattermost, please join the [Mattermost localization channel to discuss](https://community.mattermost.com/core/channels/localization). Below are descriptions of the overall localization process.
```

Comment (justinegeffen) [reply contributors/contributors/localization.md]: Thanks, Amy, it's fixed. :)

**Review by jasonblais — APPROVED** [review]

**Review by amyblais — APPROVED** [review]

**Pull request merged**